projects/plugins/jetpack/_inc/client/security/waf.jsx

```diff
@@ -495,6 +497,18 @@ export const Waf = class extends Component {
 				hideButton={ true }
 			>
 				{ isWafActive && <QueryWafSettings /> }
+				<SimpleNotice
+					showDismiss={ false }
+					status="is-info"
+					text={ __(
+						'The settings for the Firewall will be moved to Jetpack Portect in Jetpack version 13.9.',
```

Comment:
Version 13.9 was mentioned in task description, but this will likely need to be updated once the actual version is confirmed.